ui-tests/src/test/resources/features/integrations/google-sheets.feature

```diff
@@ -417,3 +428,80 @@ Feature: Google Sheets Connector
+    And select "Get sheet values" integration action
+    And fill in values
+      | SpreadsheetId | 1_OLTcj_y8NwST9KHhg8etB10xr6t3TrzaFXwW2dhpXw |
+      | Range         | A1:R5000                                     |
+      | Delay         | 30000                                        |
+      | Max results   | 200                                          |
+    And fill in values by element ID
+      | delay | 30000 |
+    Then click on the "Done" button
+
+    When select the "PostgresDB" connection
+    And select "Invoke SQL" integration action
+    And fill in invoke query input with "insert INTO todo (task) VALUES(:#task)" value
+    Then click on the "Done" button
+
+    When add integration step on position "0"
+    Then select the "Split" connection
+
+    When add integration step on position "1"
+    And select the "Data Mapper" connection
+    And check visibility of data mapper ui
+    And create data mapper mappings
+      | A;B;C;D;E;F;G | task | Underscore [_] |
+    Then click on the "Done" button
+    When click on the "Publish" button
+    And set integration name "to-db"
+    And click on the "Publish" button
+    And navigate to the "Integrations" page
+    And wait until integration "to-db" gets into "Running" state
+    Then navigate to the "Home" page
+
+    When click on the "Create Integration" button to create a new integration
+    And select the "PostgresDB" connection
+    And select "Periodic SQL invocation" integration action
+    And fill in values
+      | SQL statement | SELECT * FROM todo |
+      | Period        | 10000              |
+    Then click on the "Done" button
+
+    When select the "Google Sheets" connection
+    And select "Append values to a sheet" integration action
+    And fill in values
+      | Range | A1:H1 |
+    And fill spreadsheet ID
+    Then click on the "Next" button
+
+    When add integration step on position "0"
+    And select the "Split" connection
+
+    When add integration step on position "1"
+    And select the "Aggregate" connection
+
+    When add integration step on position "1"
+    And select the "Data Mapper" connection
+    And check visibility of data mapper ui
+    And open data bucket "Properties"
+    And create data mapper mappings
+      | task | A;B;C;D;E;F;G | Underscore [_] |
+    Then click on the "Done" button
+
+    When click on the "Publish" button
+    And set integration name "from-db-to-sheets"
+    And click on the "Publish" button
+    And navigate to the "Integrations" page
+    And wait until integration "from-db-to-sheets" gets into "Running" state
+    #it takes some time to copy that big amount of messages
+    And sleep for jenkins delay or "150" seconds
+    And verify that test sheet contains values on range "A1000:E1000"
+      | 308578 | FL | ST  JOHNS COUNTY | 1377 | 1377 |
+    And verify that test sheet contains values on range "A4999:E4999"
+      | 202669 | FL | LEON COUNTY | 0 | 9270000 |
```

Comment:
eof new line missing + maybe keep 2 empty lines before new scenario like we do in other files (more or less)